recipes/assimp/5.x/patches/0007-unvendor-deps-5.3.x.patch

```diff
@@ -0,0 +1,315 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -132,12 +132,12 @@ OPTION ( ASSIMP_IGNORE_GIT_HASH
+ IF (WIN32)
+   OPTION( ASSIMP_BUILD_ZLIB
+     "Build your own zlib"
+-    ON
```

Comment:
Why are we overwriting default option values, when we can just set their value in the recipe?